app/public/js/common/favoriteSongDirective.js

```diff
@@ -23,7 +23,7 @@ app.directive('favoriteSong', function(
                     SCapiService.deleteFavorite(userId, songId)
                         .then(function(status) {
                             if ( typeof status == "object" ) {
-                                notificationFactory.success("Song removed from likes!");
+                                notificationFactory.warning("Song removed from likes!");
```

Comment:
There's no `warning` methog in `notificationFactory`... it should be `noticationFactory.warn()`.